api-samples/tabs/inspector/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Tab Inspector",
+  "description": "Utility for working with the extension tabs api",
```

Comment:
Can we say more here, including that we give you a UI for doing basically everything with tabs and windows?